### PR TITLE
Add various spelling correction for error and variants.

### PR DESCRIPTION
**Pull request by cfi-gb**
As suggested by @peternewman in https://github.com/codespell-project/codespell/pull/1890#issuecomment-817649907

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11427,6 +11431,10 @@ etherenet->Ethernet
 ethnocentricm->ethnocentrism
 ethose->those, ethos,
 etiher->either
+etroneous->erroneous
+etroneously->erroneously
+etror->error
+etrors->errors
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:11437]: Possibly also suggest terror*?

**Review by peternewman — COMMENTED** [review]
A possible improvement

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: Yes, absolutely. Thanks for the suggestion, added in a76733f24c006e89fc05d4a7a30a4908b113eb1a

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11347,6 +11347,10 @@ errror->error
 errrors->errors
 errros->errors
 errupted->erupted
+ertoneous->erroneous
+ertoneously->erroneously
+ertor->error
+ertors->errors
```
Comment (cfi-gb) [inline codespell_lib/data/dictionary.txt:11353]:
```suggestion
ertor->error, terror,
ertors->errors, terrors,
```

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: You know you can just edit the files directly as the PR author (even via the web UI if preferred), you don't have to use suggestions. Although fair enough if you find it easier.

**Review by peternewman — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: Ah, yes. Good suggestion :+1:

**Pull request merged**